scripts/get_watch.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
I could only change naming, file => according to what is getting called in `main`